### DISCUSSION TITLE
Change my environment to opty 1.4.0.dev0

### DISCUSSION THREAD

**Discussion by Peter230655** [csu-hmc/opty#219] [Q&A]
I currently have opty 1.3.0.dev0 in my environment.
If I want to change to opty 1.4.0.dev0 would the correct way be, in Anaconda Prompt, in this environment, to say
pip install opty==1.4.0.dev0?
Would I have to de-install opty 1.3.0.dev0 before?
**Thanks!**

**Comment by moorepants** [comment]
I explained how to setup a development environment with conda in your plot pull request. It is also explained in the readme.

**Reply by Peter230655** [reply]
> I explained how to setup a development environment with conda in your plot pull request. It is also explained in the readme.

I did it as you explaind in this PR and it worked very well! (Of course it took me a couple of hours to get it right...)
